event_routing_backends/processors/xapi/tests/fixtures/expected/complete_video.json

```diff
@@ -20,8 +20,9 @@
             ]
         },
         "extensions": {
-            "https://github.com/openedx/event-routing-backends/blob/master/docs/xapi-extensions/eventVersion.rst": "1.0",
-            "https://w3id.org/xapi/video/extensions/length": 195.0
+            "https://github.com/openedx/event-routing-backends/blob/master/docs/xapi-extensions/eventVersion.rst": "1.1",
```

Comment:
I would recommend to use a more explicit and xAPI compliant IRI.

Two reasons for this: 
* by defining the extension as it is, it sticks the templates to edX events which is not efficient for the xAPI world, it could discourage others to adopt the profile
* if I understand correctly, it versions the event transformer. This information could be useful to be universal. I presume other xAPI users could need to track the information of the transformation version. Hence, it could be interesting to create an extension in the OpenEdx common profile available on the ADLNET server and define it as a universal events transformer version.

WDYT?

Reply:
Yes makes it makes sense to create extension in Open edX profile.